runtime/src/bank.rs

```diff
@@ -1907,85 +1962,108 @@ impl Bank {
             return 0.0;
         }
 
-        let point_value = PointValue { rewards, points };
-
-        let mut rewards = vec![];
         // pay according to point value
-        for (vote_pubkey, (stake_group, vote_account)) in stake_delegation_accounts.iter_mut() {
-            let mut vote_account_changed = false;
-            let voters_account_pre_balance = vote_account.lamports;
-            let vote_state: VoteState = match StateMut::<VoteStateVersions>::state(vote_account) {
-                Ok(vote_state) => vote_state.convert_to_current(),
-                Err(err) => {
-                    debug!(
-                        "failed to deserialize vote account {}: {}",
-                        vote_pubkey, err
-                    );
-                    continue;
-                }
-            };
-            let commission = Some(vote_state.commission);
-
-            for (stake_pubkey, stake_account) in stake_group.iter_mut() {
-                // curry closure to add the contextual stake_pubkey
-                let mut reward_calc_tracer = reward_calc_tracer.as_mut().map(|outer| {
```

Comment:
@ryoqun is it ok to take this out for v1.7 or should we keep it in?

Reply:
yeah, reward_calc_tracer isn't critical at all. it's just my toy..